src/main/java/com/rcv/ContestConfig.java

```diff
@@ -59,6 +58,65 @@
   // whether or not there are any validation errors
   private boolean isValid;
 
+  // function: loadContestConfig
+  // purpose: factory method to create ContestConfig from configPath
+  // - create rawContestConfig from file - can fail for IO issues or invalid json
+  // - validate rawContestConfig - can fail if certain elements do not exist
+  // - if the above succeed create and return ContestConfig wrapping rawContestConfig (cannot fail)
+  // returns: new ContestConfig object if checks pass otherwise null
+  static ContestConfig loadContestConfig(String configPath) {
+    if (configPath == null) {
+      Logger.log(Level.SEVERE, "No config path specified!");
+      return null;
+    }
+    // config will hold the new ContestConfig if construction succeeds
+    ContestConfig config = null;
+    // set config file parent folder as default user folder
+    FileUtils.setUserDirectory(new File(configPath).getParent());
+
+    // rawConfig holds the basic contest config data parsed from json
+    // this will be null if there is a problem loading it
+    RawContestConfig rawConfig = JsonParser.readFromFile(configPath, RawContestConfig.class);
+    if (rawConfig == null) {
+      Logger.log(Level.SEVERE, "Failed to load contest config: %s", configPath);
```

Comment:
```suggestion
      Logger.log(Level.SEVERE, "Failed to load contest config:\n%s", configPath);
```

Easier to read notifications when they're on separate lines from garbagey file paths.